src/content/ui-api/observability/grafana/custom-dashboards/index.md

```diff
@@ -0,0 +1,57 @@
+---
+linkTitle: Custom Grafana dashboards
+title: Creating custom Grafana dashboards
+description: Guide explaining how to manage custom grafana dashboards in our management cluster grafana.
+menu:
+  main:
+    identifier: uiapi-observability-grafana-custom-dashboards
+    parent: uiapi-observability-grafana
+weight: 40
+aliases:
+  - /observability/grafana/custom-dashboards
+last_review_date: 2022-09-12
+user_questions:
+  - How can customize dashboards?
+  - How can I create my own dashboards?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+__Beware__ that the access to this feature is experimental and the implementation specifics can change at a later time (e.g. grafana operator)
+
+As explained in our guide explaining [how to access Grafana]({{< relref "/ui-api/observability/grafana/access" >}}), we provide a subset of dashboards accessible by our customers.
+
+Sometimes, the dashboards we provide are not enough for your use case so we offer you the capability of creating your own (e.g. you are deploying your own apps on the management cluster, you want custom dashboard for your clusters).
+
+## Creating your own dashboard
+
+To create your own dashboard, you can create a configmap in the management cluster in any namespace you want (preferably the one you use for automation) and create a configmap with containing the dashboard
```

Comment:
```suggestion
To create your own dashboard, you can create a configmap in the management cluster in any namespace you want (preferably the one you use for automation) and create a configmap containing the dashboard. For example:
```

Reply:
This would also be a good place to link to any upstream docs about loading dashboards from ConfigMaps, if they exist